src/sentry/relay/config/__init__.py

```diff
@@ -66,6 +66,7 @@
     "organizations:session-replay-recording-scrubbing",
     "organizations:device-class-synthesis",
     "organizations:custom-metrics",
+    "organizations:custom-metrics-killswitch",
```

Comment:
Instead of adding a new feature, we should just remove the `"organizations:custom-metrics"` if the killswitch is enabled.

Reply:
Yeah that is a good idea indeed, will not need any relay change.